### ISSUE TITLE
Task - Filter Sources -> Text attributes of BibTex should not be indented

### ISSUE THREAD

**Issue by mjy** [SpeciesFileGroup/taxonworks#1658]
I think, technically not working, and not legal format (e.g. not recognized by Endnote filter).

**Comment by mjy** [comment]
This:
 
```
{
foo: abc
}
```

not:
 
```
{
   foo: abc
}
```

**Comment by mjy** [comment]
This might be an issue with Endnote import.  I can't image the `to_s` method from the core library failing to generate correctly.

**Comment by jlpereira** [comment]
Yep, I been looking the same at the library repository. I don't see a way to customize it there

**Comment by mjy** [comment]
You can't.  It's injecting the spaces.  I wonder if something else in Endnote is broken, there shouldn't be a problem I don't think.  Holding off on this for now.
